src/pymovements/events/engbert.py

```diff
@@ -48,6 +50,8 @@ def microsaccades(
 
     Parameters
     ----------
+    positions : np.ndarray, shape (N, 2)
+        x and y positions of N samples in chronological order
```

Comment:
so we only target one eye?

Reply:
I think we had this discussion already a few times, but lets keep it in #28 in the future, where we can decide on how to support binocular event detection one day.

just a basic summary of our past discussion outcomes:
* all our implemented event detection algorithms (and also the ones which we plan to implement) are only defined for a single eye
* we want to implement this functionality in a binocular filter in the preprocessing pipeline #28
* there we can also support different kinds of methods for merging the two eyes
* until then, the event detection will be purely monocular